pandas/core/indexes/datetimes.py

```diff
@@ -1765,6 +1768,15 @@ def bdate_range(start=None, end=None, periods=None, freq='B', tz=None,
     To learn more about the frequency strings, please see `this link
     <http://pandas.pydata.org/pandas-docs/stable/timeseries.html#offset-aliases>`__.
 
+    Examples
+    --------
+    Note how the two weekend days are skipped in the result.
+
+    >>> pd.bdate_range(start='1/1/2018', end='1/08/2018')
+    DatetimeIndex(['2018-01-01', '2018-01-02', '2018-01-03', '2018-01-04',
+               '2018-01-05', '2018-01-08'],
+              dtype='datetime64[ns]', freq='B')
```

Comment:
Hmmm...that output formatting of the lines is a little weird there.  Should they be more aligned?